ja/recurrent-neural-networks.md

```diff
@@ -0,0 +1,677 @@
+
+&#10230;[過去情報はどのくらい重要ですか？ 前の情報を削除しますか？、セルを消去しますか？　しませんか？　セルを表示するコストはどのくらいですか？]
+
+<br>
+
+
+**35. [LSTM, GRU]**
+
+&#10230;[LSTM GRU]
+
+<br>
+
+
+**36. GRU/LSTM ― Gated Recurrent Unit (GRU) and Long Short-Term Memory units (LSTM) deal with the vanishing gradient problem encountered by traditional RNNs, with LSTM being a generalization of GRU. Below is a table summing up the characterizing equations of each architecture:**
+
+&#10230;GRU/LSTM -  ゲートリカレントユニット（GRU）およびロングショートタームメモリユニット（LSTM）は、従来のRNNで問題になった勾配消失問題を解決します。LSTMはGRUの一般化名称です。以下は、各アーキテクチャの特性式をまとめた表です。
+
+<br>
+
+
+**37. [Characterization, Gated Recurrent Unit (GRU), Long Short-Term Memory (LSTM), Dependencies]**
+
+&#10230;特性評価、ゲートリカレントユニット（GRU）、ロングショートタームメモリ（LSTM）、依存関係
+
+<br>
+
+
+**38. Remark: the sign ⋆ denotes the element-wise multiplication between two vectors.**
+
+&#10230;備考：記号*は2つのベクトル間の要素ごとの乗算を表します。
+
+<br>
+
+
+**39. Variants of RNNs ― The table below sums up the other commonly used RNN architectures:**
+
+&#10230;RNNの変化版 - 以下の表は、一般的に使用されている他のRNNアーキテクチャをまとめたものです。
+
+<br>
+
+
+**40. [Bidirectional (BRNN), Deep (DRNN)]**
+
+&#10230;[双方向(BRNN)、ディープ(DRNN)] 
+
+<br>
+
+
+**41. Learning word representation**
+
+&#10230;単語表現の学習
+
+<br>
+
+
+**42. In this section, we note V the vocabulary and |V| its size.**
+
+&#10230;この節では、Vを語彙、そして|V|を語彙のサイズとして定義します。
+
+<br>
+
+
+**43. Motivation and notations**
+
+&#10230;動機と表記
+
+<br>
+
+
+**44. Representation techniques ― The two main ways of representing words are summed up in the table below:**
+
+&#10230;表現のテクニック - 単語を表現する2つの主な方法は、以下の表にまとめられています。
+
+<br>
+
+
+**45. [1-hot representation, Word embedding]**
+
+&#10230;[1-ホット表現、Wordの埋め込み]
+
+<br>
+
+
+**46. [teddy bear, book, soft]**
+
+&#10230;テディベア、本、ソフト
+
+<br>
+
+
+**47. [Noted ow, Naive approach, no similarity information, Noted ew, Takes into account words similarity]**
+
+&#10230;[表記 ow、ナイーブベイズアプローチ、類似性情報なし、表記 ew、単語の類似性を考慮に入れる]
+
+<br>
+
+
+**48. Embedding matrix ― For a given word w, the embedding matrix E is a matrix that maps its 1-hot representation ow to its embedding ew as follows:**
+
+&#10230;埋め込み行列 - 与えられた単語wに対して、埋め込み行列Eは、1-hot表現owを埋め込み行列ewに写像します。式は以下のようになります。
+
+<br>
+
+
+**49. Remark: learning the embedding matrix can be done using target/context likelihood models.**
+
+&#10230;注：埋め込み行列は、ターゲット/コンテキスト尤度モデルを使用して学習できます。
+
+<br>
+
+
+**50. Word embeddings**
+
+&#10230;単語の埋め込み
+
+<br>
+
+
+**51. Word2vec ― Word2vec is a framework aimed at learning word embeddings by estimating the likelihood that a given word is surrounded by other words. Popular models include skip-gram, negative sampling and CBOW.**
+
+&#10230;Word2vec -  Word2vecは、ある単語が他の周辺単語から導きだされる可能性を推定することで、単語の埋め込みの重みを学習することを目的としたフレームワークです。この一般的なモデルは、スキップグラム、ネガティブサンプリング、およびCBOWがあります。
+
+<br>
+
+
+**52. [A cute teddy bear is reading, teddy bear, soft, Persian poetry, art]**
+
+&#10230;[かわいいテディベアが読んでいる、テディベア、ソフト、ペルシャ詩、芸術]
```

Comment:
```suggestion
&#10230;[かわいいテディベアが読んでいる、テディベア、柔らかい、ペルシャ詩、芸術]
```
Keep this example sentence as it is in English since the order of words changes in Japanese?
日本語にすると語順が変わってしまうので例文は英語のままにしておく？